sdk/communication/Azure.Communication.Administration/src/Models/PhoneNumberReservationPurchaseOperation.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.Communication.Administration.Models
+{
+    /// <summary>
+    /// Represents a long-running phone number reservation purchase operation.
+    /// </summary>
+    public class PhoneNumberReservationPurchaseOperation : Operation<ReservationStatus>
```

Comment:
The same comments apply here as in the other Operation, I'll not repeat all of them here but add some shortened ones without copying explanation :-)